.github/workflows/release-azure-pipelines.yml

```diff
@@ -0,0 +1,103 @@
+# This pipeline will be triggered when either main branch is pushed or 2AM on workdays.
+variables:
+- name: tags
+  value: "nonproduction"

```

Comment:
Are we temporarily calling this a non-prod pipeline?

Reply:
Great question! I am intentionally configuring this as non-production for now, while we are still experimenting with it. That's because I don't want to accidentally trigger any S360 KPIs while we're drafting up the final version of the pipeline. Once we have it closer to completion, I intend to switch it to production (and use the Official pipeline template)